src/Microsoft.Health.Fhir.Web/Startup.cs

```diff
@@ -21,9 +23,9 @@ public Startup(IConfiguration configuration)
         // This method gets called by the runtime. Use this method to add services to the container.
         public virtual void ConfigureServices(IServiceCollection services)
         {
-            services.AddFhirServer(Configuration).AddCosmosDb();
+            services.AddCosmosDb().AddCosmosControlPlane(Configuration).AddDevelopmentIdentityProvider(Configuration);
```

Comment:
This should be refactored to automatically register the appropriate base objects if either the control plane cosmos db or the fhir server cosmos db is added.